onfido-java/src/main/java/com/onfido/managers/DocumentManager.java

```diff
@@ -26,24 +26,22 @@ public DocumentManager(Config config) {
 
     public Document upload(InputStream inputStream, String fileName, Document.Request request) throws IOException, OnfidoException {
 
-        RequestBody requestBody = new MultipartBody.Builder().setType(MultipartBody.FORM)
+        MultipartBody.Builder builder = new MultipartBody.Builder().setType(MultipartBody.FORM);
 
-                .addFormDataPart("applicant_id", request.getApplicantId())
-                .addFormDataPart("type", request.getType())
-                .addFormDataPart("side", request.getSide())
-                .addFormDataPart("issuing_country", request.getIssuingCountry())
+                addFormDataParam(builder, "applicant_id", request.getApplicantId());
```

Comment:
Could you fix up the indentation here and in `LivePhotoManager`? Should all be in line with `MultipartBody.Builder builder...` now